terasoluna-tourreservation-domain/src/main/resources/META-INF/spring/terasoluna-tourreservation-codelist.xml

```diff
@@ -47,7 +52,8 @@
     <entry key="6" value="5泊以下" />
   </util:map>
 
-  <bean id="CL_TOUR_DAYS" class="org.terasoluna.gfw.common.codelist.i18n.SimpleI18nCodeList">
+  <bean id="CL_TOUR_DAYS" class="org.terasoluna.gfw.common.codelist.i18n.SimpleI18nCodeList"
```

Comment:
- [ ] No need class attribute. It's already specified on abstract parent bean.

Reply:
Modified by 7e5b92654f051a6834c107b035222520946c8746